openlibrary/templates/covers/book_cover.html

```diff
@@ -26,14 +26,14 @@
             </div>
     </div>
 $else:
-    <img src="$(cover_url or '/images/icons/avatar_book-sm.png')" height="58" title="$title" alt="We need a book cover for: $title"/>
+    <img src="$(cover_url or '/images/icons/avatar_book-sm.png')" srcset="cover_lg 2x" height="58" title="$title" alt="We need a book cover for: $title"/>
```

Comment:
This might be a bit too much of a jump; from a small to a large. We could set this to med, but I think it's ok as is. Still need that `$` though :P 

```suggestion
    <img src="$(cover_url or '/images/icons/avatar_book-sm.png')" srcset="$cover_lg 2x" height="58" title="$title" alt="We need a book cover for: $title"/>
```